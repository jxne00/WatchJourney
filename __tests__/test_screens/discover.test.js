import React from 'react';
import renderer from 'react-test-renderer';

import DiscoverScreen from '../../screens/discover/Discover';

describe('tests for Discover screen', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<DiscoverScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('has 1 child', () => {
    const tree = renderer.create(<DiscoverScreen />).toJSON();
    expect(tree.children.length).toBe(1);
  });
});


