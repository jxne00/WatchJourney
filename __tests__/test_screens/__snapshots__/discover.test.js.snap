// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for Discover screen renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#74369D",
      "flex": 1,
      "padding": 10,
    }
  }
>
  <Text
    style={
      {
        "fontFamily": "Poppins-SemiBold",
        "fontSize": 24,
        "marginBottom": 10,
      }
    }
  >
    Movies
  </Text>
</View>
`;
